app/src/main/java/net/squanchy/eventdetails/EventDetailsActivity.kt

```diff
@@ -55,6 +58,9 @@ class EventDetailsActivity : AppCompatActivity() {
 
     override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
         if (requestCode == REQUEST_CODE_SIGNIN) {
+            if (resultCode == Activity.RESULT_OK) {
```

Comment:
I'd rather do it in one place, the signin activity, rather than everywhere else. ⚠️ seems less error prone and more SRP-y

Reply:
I thought about that, wasn't entirely convinced it would have been better, but I'll give it a try

Reply:
I'd rather require passing a signing origin as a parameter (which is explicit and can't _not_ be done) than rely on the implicit knowledge that you need to watch for the signin result, check if it's ok, then know you need to track it (which incidentally requires you to inject the tracker everywhere) and _how_ to track it. Reacting to an activity result is not enforceable, too, which is my main worry